internal/communicator/ssh/communicator.go

```diff
@@ -816,3 +825,15 @@ func (c *bastionConn) Close() error {
 	c.Conn.Close()
 	return c.Bastion.Close()
 }
+
+func QuoteShell(args []string, targetPlatform string) (string, error) {
```

Comment:
This function doesn't need to be used outside of this package, so I think we should unexport it by renaming to `quoteShell`.